python/lsst/daf/persistence/butlerLocation.py

```diff
@@ -89,24 +87,23 @@ def __init__(self, datasetType, obj, setter, getter, subset, inputOnly):
 
         def __repr__(self):
             return 'ComponentInfo(datasetType:%s, obj:%s, setter:%s, getter:%s, subset:%s)' % \
-                    (self.datasetType, self.obj, self.setter, self.getter, self.subset)
-
+                (self.datasetType, self.obj, self.setter, self.getter, self.subset)
 
     def __repr__(self):
-        return 'ButlerComposite(assembler:%s, disassembler:%s, python:%s, dataId:%s, mapper:%s, componentInfo:%s, repository:%s)' % \
-                (self.assembler,
-                self.disassembler,
-                self.python,
-                self.dataId,
-                self.mapper,
-                self.componentInfo,
-                self.repository)
+        return 'ButlerComposite(assembler:%s, disassembler:%s, python:%s, dataId:%s, mapper:%s, ' \
+            'componentInfo:%s, repository:%s)' % \
+            (self.assembler,
+             self.disassembler,
+             self.python,
+             self.dataId,
+             self.mapper,
+             self.componentInfo,
+             self.repository)
 
         def __repr__(self):
             return "ComponentInfo(datasetType=%s, obj=%s, setter=%s, getter=%s)" % (
                 self.datasetType, self.obj, self.setter, self.getter)
```

Comment:
this is the definition to remove.

Reply:
I think there's another one at line 146...